apollo-router/src/spec/schema.rs

```diff
@@ -430,8 +431,8 @@ impl Schema {
 }
 
 impl Schema {
-    /// Extracts a string slice containing the entire [`Schema`].
-    pub fn as_str(&self) -> &str {
+    /// Extracts a string containing the entire [`Schema`].
+    pub fn as_string(&self) -> &Arc<String> {
```

Comment:
Instead of return a reference to the Arc, why not have this function return a clone of the Arc and then API consumers don't have to decide whether to clone or not?

Reply:
If a user needs to clone the Arc they’ll know soon enough. I don’t see this as an improvement but don’t feel strongly about it either.

Reply:
Ok. Leave as is.